roles/dtc/common/templates/ndfc_inventory.j2

```diff
@@ -14,4 +14,15 @@
   max_hops: 0 # this is the default value as it is not defined into the data model
   role: {{ switch['role'] }}
   preserve_config: false
+{% if switch['poap'] is defined %}
```

Comment:
We will need to refactor this after https://github.com/netascode/ansible-dc-vxlan/pull/204 is merged

Reply:
this should be done now too